doc/types.md

```diff
@@ -72,6 +74,8 @@ Examples:
 ```js
 "#f00"
 "#ff0000"
+"#ff00007c" //50% opacity red
+"#ff06" //60% opacity yellow
```

Comment:
`6` should be equivalent to `66`, which represents 40%. Please fix the documentation.
See http://stackoverflow.com/questions/15852122/hex-transparency-in-colors